modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -53,11 +61,32 @@ You can use a Mule expression to instruct the service to look for the IP address
 #[message.inboundProperties['http.headers']['X-Forwarded-For']]
 ----
 
-Insert the IP address in the IP expression field:
+When you insert the IP address in the IP expression field of the policy parameters, the API Platform is instructed to look at the first IP address that ranks in the concatenated values of the `x-Forwarded-For` header of the request:
 
 image::ip-whitelist-ip-expression.png[ip+blacklist+ip_expression]
 
-This instructs API Platform to look at the first IP address that ranks in the concatenated values of the 'x-Forwarded-For' header of the request.
+
+== Configuring Policy Parameters
+
+When you apply the IP Blacklist policy to your API from the UI, you can configure the following parameters:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Parameter | Description 
+|IP expression  
+|The Mule expression to be used for extracting the IP address from this API request. 
```

Comment:
In Mule 4 is a DataWeave 2.0 expression, which is quite similar to MEL in this case, but not the same.

Reply:
Done! Thanks!